_docs/_latest/configure/scanner/default-rules.md

```diff
@@ -12,6 +12,11 @@ resources.
 
 ---
 
+## Audit logging
```

Comment:
I don't think there are any default rules for audit logging. This scanner is disabled by default and there are [no default rules](https://github.com/forseti-security/terraform-google-forseti/blob/master/modules/rules/templates/rules/audit_logging_rules.yaml).